### PR TITLE
Release

### LINKED ISSUE #1: Release first version

### LINKED ISSUE #8: Add Badges to Readme
[DOI](https://zenodo.org/badge/latestdoi/335652633)
[License: MPL 2.0](https://opensource.org/licenses/MPL-2.0)
[GitHub release](https://github.com/fhooeaist/imaging/releases)
[javadoc](https://javadoc.io/doc/science.aist.imaging/imaging) 
[Maven Central](https://search.maven.org/search?q=g:"science.aist.imaging")

### LINKED ISSUE #9: Fix Javadoc Generation
The project is built using java11+, which means that everything that is not within a module gets assign to the default module. This causes the javadoc-search to work incorrectly.

```xml
...
<configuration>
    ...
    <additionalJOptions>
        <additionalJOption>--no-module-directories</additionalJOption>
    </additionalJOptions>
   ...
</configuration>
...

### LINKED ISSUE #10: [BUG] Non-required dependencies
**Describe the bug**
There are multiple non-required maven dependencies left over from the open sourcing process. 

**To Reproduce**
E.g. Look into API pom.xml

**Expected behavior**
Removed dependencies.

### PR DESCRIPTION
**Pull request by pointan**
Closes #1 
Closes #9 
Closes #10 
Closes #8

### CODE REVIEW

**Review by cpraschl — APPROVED** [review]

File: api/pom.xml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+        <dependency>
+            <groupId>org.hibernate.javax.persistence</groupId>
+            <artifactId>hibernate-jpa-2.1-api</artifactId>
+            <version>${hibernate-jpa.version}</version>
+        </dependency>
+        <dependency>
+            <groupId>org.hibernate</groupId>
+            <artifactId>hibernate-core</artifactId>
+            <version>${hibernate.version}</version>
+        </dependency>
```
Comment (2er0) [inline api/pom.xml:30]: Remove those dependencies - are they required?

**Review by 2er0 — COMMENTED** [review]
CleanUp Hibernate

File: api/pom.xml (hunk 1 above)
Comment (pointan) [reply api/pom.xml]: Yes, removed.

**Comment by pointan** [comment]
I also added #8, #9 and #10 to this MR.

**Review by 2er0 — APPROVED** [review]

**Pull request merged**